Fix test_scorecard.py with pandas-1.5.1

## Describe the Problem ##
Running the pytests on WCOSS2 Acorn produced this error:
```
================================================================================= ERRORS =================================================================================
___________________________________________________________________ ERROR collecting test_scorecard.py ___________________________________________________________________
ImportError while importing test module '/apps/sw_review/emc/METcalcpy/METcalcpy-2.0.0/test/test_scorecard.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/apps/spack/python/3.8.6/intel/19.1.3.304/pjn2nzkjvqgmjw4hmyz43v5x4jbxjzpk/lib/python3.8/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
test_scorecard.py:4: in <module>
    from metcalcpy.scorecard import Scorecard, pd
../metcalcpy/scorecard.py:47: in <module>
    from pandas.core.common import SettingWithCopyWarning
E   ImportError: cannot import name 'SettingWithCopyWarning' from 'pandas.core.common' (/u/julie.prestopnik/.local/lib/python3.8/site-packages/pandas/core/common.py)
======================================================================== short test summary info =========================================================================
ERROR test_scorecard.py
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
============================================================================ 1 error in 2.23s ============================================================================
```

### Expected Behavior ###
This pytest should pass.

### Environment ###
Describe your runtime environment:
1. Machine: WCOSS2 Acorn
2. OS: SUSE Linux Enterprise Server 15 SP3, Linux alogin01 5.3.18-150300.59.90-default
3. Software version number(s): METcalcpy-2.0.0

### To Reproduce ###
Describe the steps to reproduce the behavior:
*1. Go to '...'*
*2. Click on '....'*
*3. Scroll down to '....'*
*4. See error*
*Post relevant sample data following these instructions:*
*https://dtcenter.org/community-code/model-evaluation-tools-met/met-help-desk#ftp*
Ran _python3 -m pytest_  in /apps/sw_review/emc/METcalcpy/METcalcpy-2.0.0/test

### Relevant Deadlines ###
ASAP

### Funding Source ###
2792542, 2773542

## Define the Metadata ##

### Assignee ###
- [x] Select **engineer(s)** or **no engineer** required
- [ ] Select **scientist(s)** or **no scientist** required

### Labels ###
- [x] Select **component(s)**
- [x] Select **priority**
- [x] Select **requestor(s)**

### Projects and Milestone ###
- [x] Select **Organization** level **Project** for support of the current coordinated release
- [x] Select **Repository** level **Project** for development toward the next official release or add **alert: NEED PROJECT ASSIGNMENT** label
- [x] Select **Milestone** as the next bugfix version

## Define Related Issue(s) ##
Consider the impact to the other METplus components.
- [ ] [METplus](https://github.com/dtcenter/METplus/issues/new/choose), [MET](https://github.com/dtcenter/MET/issues/new/choose), [METdataio](https://github.com/dtcenter/METdataio/issues/new/choose), [METviewer](https://github.com/dtcenter/METviewer/issues/new/choose), [METexpress](https://github.com/dtcenter/METexpress/issues/new/choose), [METcalcpy](https://github.com/dtcenter/METcalcpy/issues/new/choose), [METplotpy](https://github.com/dtcenter/METplotpy/issues/new/choose)

## Bugfix Checklist ##
See the [METplus Workflow](https://metplus.readthedocs.io/en/latest/Contributors_Guide/github_workflow.html) for details.
- [ ] Complete the issue definition above, including the **Time Estimate** and **Funding Source**.
- [ ] Fork this repository or create a branch of **main_\<Version>**.
Branch name: `bugfix_<Issue Number>_main_<Version>_<Description>`
- [ ] Fix the bug and test your changes.
- [ ] Add/update log messages for easier debugging.
- [ ] Add/update unit tests.
- [ ] Add/update documentation.
- [ ] Push local changes to GitHub.
- [ ] Submit a pull request to merge into **main_\<Version>**.
Pull request: `bugfix <Issue Number> main_<Version> <Description>`
- [ ] Define the pull request metadata, as permissions allow.
Select: **Reviewer(s)** and **Development** issues
Select: **Organization** level software support **Project** for the current coordinated release
Select: **Milestone** as the next bugfix version
- [ ] Iterate until the reviewer(s) accept and merge your changes.
- [ ] Delete your fork or branch.
- [ ] Complete the steps above to fix the bug on the **develop** branch.
Branch name:  `bugfix_<Issue Number>_develop_<Description>`
Pull request: `bugfix <Issue Number> develop <Description>`
Select: **Reviewer(s)** and **Development** issues
Select: **Repository** level development cycle **Project** for the next official release
Select: **Milestone** as the next official version
- [ ] Close this issue.

--- a/docs/Users_Guide/release-notes.rst
+++ b/docs/Users_Guide/release-notes.rst
@@ -49,21 +49,21 @@ METcalcpy Version 2.0.0 release notes (20221207)
 
    * **Create checksum for released code** (`#209 <https://github.com/dtcenter/METcalcpy/issues/209>`_)
 
-   * **Add modulefiles used for installation on various machines (`#204 <https://github.com/dtcenter/METcalcpy/issues/204>`_)
+   * **Add modulefiles used for installation on various machines** (`#204 <https://github.com/dtcenter/METcalcpy/issues/204>`_)
 
-   * **Identify minimal "bare-bones" Python packages to inform operational installation (`#152 <https://github.com/dtcenter/METcalcpy/issues/152>`_)
+   * **Identify minimal "bare-bones" Python packages to inform operational installation** (`#152 <https://github.com/dtcenter/METcalcpy/issues/152>`_)
 
-   * **Convert scorecard.R_tmp to Python (`#179 <https://github.com/dtcenter/METcalcpy/issues/179>`_)
+   * **Convert scorecard.R_tmp to Python** (`#179 <https://github.com/dtcenter/METcalcpy/issues/179>`_)
 
 * Bugfixes:
 
-   * **Vertical Interpolation DimensionalityError in migrating from Python 3.7.10 to Python 3.8.6 (`#180 <https://github.com/dtcenter/METcalcpy/issues/180>`_)
+   * **Vertical Interpolation DimensionalityError in migrating from Python 3.7.10 to Python 3.8.6** (`#180 <https://github.com/dtcenter/METcalcpy/issues/180>`_)
 
-   * **Address Github Dependabot Issues (`#193 <https://github.com/dtcenter/METcalcpy/issues/193>`_)
+   * **Address Github Dependabot Issues** (`#193 <https://github.com/dtcenter/METcalcpy/issues/193>`_)
 
-   * **Deprecation and other warnings in event_equalize.py and other modules (`#153 <https://github.com/dtcenter/METcalcpy/issues/153>`_)
+   * **Deprecation and other warnings in event_equalize.py and other modules** (`#153 <https://github.com/dtcenter/METcalcpy/issues/153>`_)
 
-   * **Some METcalcpy tests fail with Python 3.8 and upgraded packages (`#154 <https://github.com/dtcenter/METcalcpy/issues/154>`_)
+   * **Some METcalcpy tests fail with Python 3.8 and upgraded packages** (`#154 <https://github.com/dtcenter/METcalcpy/issues/154>`_)
     
 METcalcpy Upgrade Instructions
 ==============================

--- a/metcalcpy/scorecard.py
+++ b/metcalcpy/scorecard.py
@@ -44,7 +44,6 @@ import warnings
 import math
 import numpy as np
 from pandas import DataFrame
-from pandas.core.common import SettingWithCopyWarning
 
 from metcalcpy import GROUP_SEPARATOR, DATE_TIME_REGEX
 from metcalcpy.util.tost_paired import pt
@@ -316,7 +315,7 @@ class Scorecard:
             for component in derived_curve_component.first_component:
                 if component == fcst_lead:
                     with warnings.catch_warnings():
-                        warnings.simplefilter(action="ignore", category=SettingWithCopyWarning)
+                        warnings.simplefilter(action="ignore", category=Warning)
                         df.at[0, 'fcst_lead'] = fcst_lead
                     break
 

